docker/Dockerfile

```diff
@@ -1,12 +1,104 @@
-FROM amd64/debian:stable-slim
-ENV GOTIFY_SERVER_PORT="80"
+ARG BUILDKIT_SBOM_SCAN_CONTEXT=true
```

Comment:
I like having this all in one Dockerfile, but I dislike that the binaries for the linux platforms are built twice (one for the GH release, one for docker). I'd really like that the binaries for the same arch/os are the same for docker and the binary release.

This is probably a nice to have, as I don't really see a good / easy way for this. The binary could be copied from the built image, but this also feels a little weird.

Reply:
You can build the image, create a dummy container and `docker cp` it out I suppose?  I agree your behavior is better. I think this is the easiest fix?

Reply:
Yeah seems better. It would probably be cool we don't have to built the full multiarch image for this. E.g. I run build-bin-arm64, then it only runs with --platform=linux/arm64. But we should probably wait until someone requests this, as I don't think many need this feature.

Reply:
I wrote an internal recipe to keep the build commands and LD_FLAGS the same. I think it is fine to just build twice.